packages/cli/src/aws/update-server.test.ts

```diff
@@ -110,4 +151,10 @@ describe('update-server command', () => {
     await main(medplum, ['node', 'index.js', 'aws', 'update-server', 'incomplete']);
     expect(console.log).toBeCalledWith('ECS Cluster not found');
   });
+
+  test('Update server stack missing service', async () => {
```

Comment:
test coverage for https://sonarcloud.io/code?id=medplum_medplum&pullRequest=1944&selected=medplum_medplum%3Apackages%2Fcli%2Fsrc%2Faws%2Fupdate-server.ts&line=21

Reply:
also this covers:
https://sonarcloud.io/code?id=medplum_medplum&pullRequest=1944&selected=medplum_medplum%3Apackages%2Fcli%2Fsrc%2Faws%2Futils.ts&line=131